resources/styles/book-content/tips-for-success.less

```diff
@@ -16,25 +18,43 @@
     padding: 0 10px;
     margin: 0;
   }
-  > ul:first-child {
-    padding-top: 10px;
-    border-top: 1px solid @tutor-neutral-light;
+  p {
+    #fonts > .sans(1.7rem, 1.5em);
+    color: @tutor-neutral-dark;
+    padding: 15px 20px;
+    border-left: 1px solid @tutor-neutral-lighter;
+    border-right: 1px solid @tutor-neutral-lighter;
+    background: @tutor-neutral-lightest;
+    &:first-child {
+      border-top: 1px solid @tutor-neutral-lighter;
+    }
+    &:last-child {
+      border-bottom: 1px solid @tutor-neutral-lighter;
+    }
+  }
+  > ul {
+    #fonts > .sans(1.7rem, 1.5em);
+    color: @tutor-neutral-dark;
+    padding: 10px;
+    border-left: 1px solid @tutor-neutral-lighter;
+    border-right: 1px solid @tutor-neutral-lighter;
     list-style: none;
-    
+    background: @tutor-neutral-lightest;
+    &:first-child {
+    border-top: 1px solid @tutor-neutral-light;
```

Comment:
Can you fix the indentation of this? Also, if this pattern is common would it warrant a mixin that takes the color as an arg?